crossing-repair.lic

```diff
@@ -62,7 +62,7 @@ class CrossingRepair
     amount, denom = settings.sell_loot_money_on_hand.split(' ')
     @keep_copper = DRCM.convert_to_copper(amount, denom)
     @repair_withdrawal_amount = settings.repair_withdrawal_amount
-    @hometown_data = get_data('town')[settings.hometown]
+    @hometown_data = get_data('town')[settings.force_repair_town] || get_data('town')[settings.hometown]
```

Comment:
💡 This reminds me that [sell-loot](https://github.com/rpherbig/dr-scripts/blob/6531ea5aea2c4c47013a5c3465c333516531733a/sell-loot.lic#L39) does something similar and that perhaps we should follow that model. In which case [line 20](https://github.com/rpherbig/dr-scripts/pull/5623/files#diff-c6a8fe5c06adac610cb24575dd42aa478124886363fd89b715026f2070e03a64R20) in crossing-repair is where the consideration of the `force_repair_town` would take place.

```ruby
town = DRC.get_town_name(args.town || settings.force_repair_town)
```

Doing it this way also means that the values specified in `force_repair_town` can be anything that `DRC.get_town_name` supports (official and colloquial names).